docs/docusaurus/docs/core/configure_project_settings/configure_data_docs/_backends/_s3.md

```diff
@@ -0,0 +1,21 @@
+An Amazon S3 Data Doc site requires the following `store_backend` information:
+
+- `class_name`: The name of the class to implement for accessing the target environment.  For Amazon S3 this will be `TupleS3StoreBackend`.
+- `bucket`: The name of the Amazon S3 bucket that will host the Data Docs site.
+- `prefix`: The path of the folder that will contain the Data Docs pages relative to the root of the Amazon S3 bucket.  The combination of `container` and `bucket` must be unique accross all Stores used by a Data Context.
+- `boto3_options`: The credentials for accessing your Amazon S3 account.  Amazon S3 supports multiple methods of providing credentials, such as use of an endpoint url, access key, or role assignment.  For more information on how to configure your Amazon S3 credentials, see Amazon's documentation for how to [Configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
```

Comment:
I think as long as we link to aws docs, we're good here. I haven't tried too many options with boto3_options though.